app/src/components/Header.tsx

```diff
@@ -1,29 +1,16 @@
 import React from 'react';
-import { Platform, StyleSheet, Text, View } from 'react-native';
-
-const styles = StyleSheet.create({
-  android: {
-    top: 35,
```

Comment:
Don't remove this, it's necessary for android builds

Reply:
I've added the CSS style directly on the JSX via NativeWind CSS (android:top-9 - it's 36px), I think it's a better practice.

I we can change the 36px later if the amount is too much or too little

Reply:
Deal! The StyleSheet uses density pixels, so it scale well for other devices. In the way that you explained, it seems that does the same effect. I'll approve and merge tomorrow